src/main/java/org/ibp/api/rest/dataset/DatasetResource.java

```diff
@@ -475,4 +475,18 @@ private DatasetExportService getExportFileStrategy(final String fileType) {
 
 		return new ResponseEntity<>(this.studyDatasetService.getObservationUnitsMetadata(studyId, datasetId, request), HttpStatus.OK);
 	}
+
+	@ApiOperation(value = "Count observation-unit of dataset", notes = "Returns count of observation unit of dataset")
```

Comment:
value = "Count observation units of dataset"
notes = "Returns count of observation units of dataset"